hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/helpers/BlockUtils.java

```diff
@@ -237,4 +237,21 @@ public static void verifyBCSId(KeyValueContainer container, BlockID blockID)
               + containerBCSId + ".", UNKNOWN_BCSID);
     }
   }
+
+  /**
+   * Remove container KV metadata from per-disk db store.
+   * @param containerData
+   * @param conf
+   * @throws IOException
+   */
+  public static void dropAllFromDB(KeyValueContainerData containerData,
```

Comment:
1.  dropAllFromDB -> removeContainerFromDB
2.  removeDB(),  the logic of SCHEMA_V3 handling is not used. Better remove it.

Reply:
For 2, actually it is used when the disk failed, then the db handler should be removed.

Reply:
Oh, I think for 2 I shall remove the logic indeed, because the disk failure case is handled in other places.